spk/fdupes/Makefile

```diff
@@ -0,0 +1,23 @@
+SPK_NAME = fdupes
+SPK_VERS = 1.51
+SPK_REV = 1
+SPK_ICON = src/fdupes-logo.png
+
+DEPENDS = cross/fdupes
+
+MAINTAINER = SynoCommunity
```

Comment:
Your name here, as you're the creator and presumably will/should continue to maintain the package.